src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -308,6 +308,49 @@ The zero register is extended with zero metadata and a cleared valid tag: this i
 
 NOTE: Instruction encodings refer to *c* registers when they use `cs1`, `cd`, etc. instead of `rs1`, `rd` (which reads/writes the XLEN-bit subset of the register).
 
+==== Added CSRs
+
+{cheri_base32_ext_name} and {cheri_base64_ext_name} add the capability CSR shown in
```

Comment:
Could you add it to [#base_cap_registers]?

Reply:
How would you want to fit it into the diagram? I am under the impression that we do not want to specify anything the actual integer register (utid), but only the capability register (utidc) and then make it WARL.

Would you want utid, then immediately extend it to utidc, and then make that WARL?